benchmarks/graphics_pipeline/main.cpp

```diff
@@ -648,6 +670,12 @@ void ProjApp::CreateSpherePipelines()
                 gpCreateInfo.outputState.depthStencilFormat     = GetSwapchain()->GetDepthFormat();
                 gpCreateInfo.pPipelineInterface                 = mSphere.pipelineInterface;
                 PPX_CHECKED_CALL(GetDevice()->CreateGraphicsPipeline(&gpCreateInfo, &mPipelines[pipelineIndex++]));
+
+                // Position Planar Pipeline
+                gpCreateInfo.vertexInputState.bindingCount = 2;
```

Comment:
Something surprising, but maybe it's by design (@wangra-google for this?)
- The interleaved representation makes sense to me.
- The linear representation seems half-linear, half-interleaved:
 position & rest have their own linear spaces.
 tangent/uv/normal are interleaved in the second space.

Is that expected?